components/Link.js

```diff
@@ -46,9 +46,14 @@ const Link = ({ ['aria-label']: ariaLabel, children, className, inline, unstyled
     dataAttrs = { 'data-white': white };
   }
 
+  let target;
+  if (isExternal) {
```

Comment:
What if instead of an explicit `isExternal` flag we just looked at `href` and determined if it's a relative path or not?